mage_ai/settings/__init__.py

```diff
@@ -27,10 +29,18 @@ def is_disable_pipeline_edit_access():
 
 DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
 
+#------------------------- DISABLE TERMINAL ----------------------#'
+
+DISABLE_TERMINAL = os.getenv('DISABLE_TERMINAL', '0').lower() in ('true', '1', 't')
```

Comment:
it seems like there are some extra changes here that aren't related to the current issue. can you remove them?